index.bs

```diff
@@ -1130,6 +1156,392 @@ The following extended attributes are applicable to interfaces:
 </div>
 
 
+<h3 id="idl-mixins">Mixins</h3>
+
+A <dfn export>mixin</dfn> is a definition (matching <emu-nt><a href="#prod-Mixin">Mixin</a></emu-nt>)
+that declares state and behavior that can be [=included=] by one or more [=interfaces=],
+and that are exposed by objects that implement an [=interface=] that [=includes=] the [=mixin=].
+
+<pre class="webidl" class="syntax">
+    mixin identifier {
+      /* mixin_members... */
+    };
+</pre>
+
+Note: Mixins, much like [=partial interfaces=], are intended for use as a specification editorial aide,
+allowing a coherent set of functionalities to be grouped together,
+and included in multiple interfaces, possibly across documents.
```

Comment:
We should forward-reference your note below about the difference between them.

Reply:
Done.